### PR TITLE
Refactoring tests for CollectionOfOne

### PR DESCRIPTION
**Pull request by frootloops**

### CODE REVIEW

File: test/1_stdlib/CollectionOfOne.swift [hunk 1]
```diff
@@ -1,36 +1,20 @@
...
-}
-print(".")
-// CHECK-NEXT: for loop: 2.
-
-let twentyOne = CollectionOfOne(21)
+let OneTests = TestSuite("CollectionOfOne")
+OneTests.test("Basic tests") {
+  for x in CollectionOfOne(2) {
+    expectEqual(2, x)
+  }
```
Comment (gribozavr) [inline test/1_stdlib/CollectionOfOne.swift]: The test used to check not only the values, but also that the loop only did one iteration -- could you add a counter as well?

**Comment by gribozavr** [comment]
@frootloops Thank you!  LGTM modulo a small comment.

**Comment by frootloops** [comment]
@gribozavr check it again please

**Comment by gribozavr** [comment]
@swift-ci Please test and merge

**Pull request merged**